### DISCUSSION TITLE
Cannot activate the group option

### DISCUSSION THREAD

**Discussion by Thenet78** [dani-garcia/vaultwarden#4246] [General]
Good morning,

I allow myself to write this message because I can not activate the group option in Vaultwarden yet it seems to me that mma configuration of my docker is good.

In the/docker/vaultwarden/vw-data/config.json part I add the line "ORG_GROUPS_ENABLED": true, then I save and restart vaultwarden.

**Comment by stefan0xC** [comment]
You really shouldn't edit the `config.json` file manually. 
Please read https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview

**Reply by Thenet78** [reply]
Yeah that’s what I did.

![Capture d'écran 2024-01-16 145214](https://github.com/dani-garcia/vaultwarden/assets/122619291/56a698fc-c9f9-40b0-87a6-523cfe487160)

**Reply by Thenet78** [reply]
I have always made the changes since configuration but I do not understand why this option does not work. it is a problem of nomentelature of the code ? or other

**Reply by stefan0xC** [reply]
> Note that config settings under the `Read-Only Config` section of the admin page can only be set via environment variables, so you must restart vaultwarden to make changes to them.

**Reply by Thenet78** [reply]
![image](https://github.com/dani-garcia/vaultwarden/assets/122619291/81700078-377d-4f2c-bea4-57318ba1547f)

**Reply by Thenet78** [reply]
I have restarted the server but still cannot enable the option.

**Reply by stefan0xC** [reply]
The read-only config section already says what you should do:
![vw-admin-ready-only](https://github.com/dani-garcia/vaultwarden/assets/509385/5fd19bf3-8415-42ca-ae55-a49d769dac60)

**Comment by Thenet78** [comment]
Hello I allow myself to restart the discussion because I am still blocked

**Reply by stefan0xC** [reply]
Since we don't recommend editing the `config.json` manually I am unwilling to be more helpful with your specific problem (or see it as something that we should fix immediately in the code).

However, I realized that the [configuration overview](https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview) was not as clear as I thought it is, so I decided to rewrite it a bit. Can you please read the page and give feedback if it now answers your question?

**Comment by BlackDex** [comment]
You need to, as is mentioned in the _NOTES_ the screenshot shows which @stefan0xC posted above, add this as an environment variable.

So, you need to set an env variable `ORG_GROUPS_ENABLED=true` in either your docker compose file, or where ever you configured all other read-only config settings like `DATABASE_URL`
